setup.py

```diff
@@ -33,6 +33,7 @@
     "pennylane-lightning>=0.33",
     "requests",
     "typing_extensions",
+    "setuptools",
```

Comment:
@mlxd will this... work? Don't you need `setuptools` to run the logic in `setup.py`?

Reply:
E.g., this file will fail at line 17
```python
from setuptools import setup, find_packages
```

Reply:
iirc this is the catch-22 that `pyproject.toml` was trying to solve -- letting you specify build dependencies independent of the build system

Reply:
The issue is for PennyLane at runtime in a virtual-env --- the module is already available for building the wheels, as the wheel package depends on it (and cibuildwheel will handle that). We need `pkg_resources` for the device loader at runtime, so we need it in any environment the wheel is installed onto.

Reply:
got it! Maybe worth including `pkg_resources` here then instead of `setuptools` to be more specific?

Reply:
Works fine in a venv with the above addition.